jmix-core/core/src/test/groovy/properties/TestCoreProperties.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2022 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package properties;
+
+import io.jmix.core.CoreProperties;
+
+import java.time.Duration;
+import java.util.List;
+
+public class TestCoreProperties extends CoreProperties {
```

Comment:
It seems that "groovy" directory is a wrong place for java class. `java/test_support/datatypes` seems to be a better choice.
Also, what is the purpose of the class? Is it for setting the `decimalValueRoundByFormat` property only? If so, then maybe we could add the `@TestPropertySource` annotations to test classes with explicit property value instead?

Reply:
Moved class to `java/test_support`, without `datatypes` sub-package - this class can be used not only for `datatype` scope of tests.
This class helps to change property values in different tests within single test class. Adding a 'static' property source for entire class looks less suitable for this case. Don't want to split it into multiple test classes based on possible values of involved properties.